config/initializers/ruby_money.rb

```diff
@@ -1,8 +1,16 @@
 # frozen_string_literal: true
 
+require 'money/bank/currencylayer_bank'
+
 Money.locale_backend = :i18n
 
-eu_bank = EuCentralBank.new
-Money.default_bank = eu_bank
+mclb = Money::Bank::CurrencylayerBank.new
+mclb.access_key = ENV.fetch("CURRENCY_LAYER_API_KEY", "")
+mclb.currencylayer = true
+mclb.ttl_in_seconds = 86_400
+mclb.cache = 'rails_money_cache.txt' # To be replaced with proper cache path.
+mclb.update_rates
```

Comment:
This would mean rates are only updated once per server boot. We sometimes go for days without redeploying, which in some cases may mean exchange rates are up to ~10% out of sync.

(See also my other comment about more dynamic updating below)

Reply:
I couldn't find a proper documentation on these part, but my best guess is that if `ttl_in_seconds` is set, whenever we ask the library to do an exchange, if the previous exchange was fetched before x seconds, it will invalidate the cache and fetch the newer rates. I feel so because if this is not how it works, it doesn't make sense to have `ttl_in_seconds`. What do you think?

Reply:
I just verified from the library's [source code](https://github.com/phlegx/money-currencylayer-bank/blob/1d4763654511a2355a0bd22db8e0fec7b2fa2a94/lib/money/bank/currencylayer_bank.rb#L160) that my guess is correct.